pkg/ccl/crosscluster/physical/standby_read_ts_poller_job_test.go

```diff
@@ -58,17 +56,15 @@ func TestStandbyReadTSPollerJob(t *testing.T) {
 	serverutils.SetClusterSetting(t, c.DestCluster, "bulkio.stream_ingestion.standby_read_ts_poll_interval", pollInterval)
 
 	registry := c.ReaderTenantServer.JobRegistry().(*jobs.Registry)
-	jobRecord := makeStandbyReadTSPollerJobRecord(registry, username.MakeSQLUsernameFromPreNormalizedString("user"))
-	sqlDB := c.ReaderTenantServer.InternalDB().(isql.DB)
 
-	err := sqlDB.Txn(ctx, func(ctx context.Context, txn isql.Txn) error {
-		_, err := registry.CreateJobWithTxn(ctx, jobRecord, jobRecord.JobID, txn)
-		return err
+	testutils.SucceedsSoon(t, func() error {
+		runningJobs := registry.CurrentlyRunningJobs()
+		if len(runningJobs) != 1 {
```

Comment:
there are lots of other jobs in this registry aren't there?

Reply:
I think I'd go back to `WaitForJobToRun`, and just change it to find the job ID based on querying for the job by job type in `crdb_internal.jobs`